apollo-compiler/src/test/graphql/com/example/operation_directive/TestOperation.graphql

```diff
@@ -0,0 +1,5 @@
+query TestQuery @myModifier {
```

Comment:
that interesting am I right that this `myModifier` directive is not declared in the schema? I thought that GraphQL spec says that directive also must be declared in the schema?

Reply:
Yup, that's 100% correct. I added this test as part of checking https://github.com/apollographql/apollo-android/issues/2661. 

But since `myModifier` is not in the schema, this means we're missing some validation there. I added https://github.com/apollographql/apollo-android/issues/2673 as a follow up.

Reply:
Yeah, I guess we missed that validation as we have tests for skip / include and I don't see them either in schema sdl.